tox.ini

```diff
@@ -51,6 +51,42 @@ deps = -rdocumentation/requirements.txt
 commands = sphinx-build -d "{toxworkdir}/docs_doctree" ./documentation/source "{toxworkdir}/docs_out" --color -bhtml {posargs}
            python -c 'import pathlib; print("documentation available under file://\{0\}".format(pathlib.Path(r"{toxworkdir}") / "docs_out" / "index.html"))'
 
+[testenv:gcov]
+setenv =
+    CXX = "ccache g++"
+    CC = "ccache gcc"
+    CFLAGS = -Werror -Wno-deprecated-declarations -g --coverage
+    LDFLAGS = --coverage
+    CXXFLAGS = -Werror
+    COCOTB_LIBRARY_COVERAGE = 1
+passenv =
+    SIM
+    TOPLEVEL_LANG
+    ALDEC_LICENSE_FILE
+    SYNOPSYS_LICENSE_FILE
+    LM_LICENSE_FILE
```

Comment:
You can mixin behavior by making gcov a tox matrix element: `blah-blah-gcov`, and only setting the necessary options there. But as I mentioned, you could probably do this without using tox.